electionleaflets/apps/core/helpers.py

```diff
@@ -41,7 +58,7 @@ def get(self, endpoint, params=None, version="next", json=True):
         url = urljoin(self.YNR_BASE, path)
         params = params or {}
         params["auth_token"] = self.API_KEY
-        resp = requests.get(url, params=params)
+        resp = session.get(url, params=params)
```

Comment:
While we are looking at making this more robust, can we explicitly set a timeout on the `.get()` here

Reply:
I've switched to using `get_form_kwargs`. This doesn't in itself solve the problem, but it does mean we don't need to rely on the structure of the `initial` kwarg. 

I've also made it blow up if not provided, so we'll catch any problem earlier.